src/openrct2/windows/dropdown.h

```diff
@@ -23,7 +23,7 @@
 #define DROPDOWN_SEPARATOR            0
 #define DROPDOWN_FORMAT_COLOUR_PICKER 0xFFFE
 #define DROPDOWN_FORMAT_LAND_PICKER   0xFFFF
-#define DROPDOWN_ITEMS_MAX_SIZE       64
+#define DROPDOWN_ITEMS_MAX_SIZE       90     // Changed from 64 to 90 to support arbritrary ride type dropdown size
```

Comment:
It seems very arbitrary. If we can support longer lists properly, why not change it to something like 256?